lib/chf/slackistrano_messaging.rb

```diff
@@ -0,0 +1,20 @@
+module Chf
+  class SlackistranoMessaging < Slackistrano::Messaging::Base
+    def channels_for(action)
+      channels = if fetch(:stage).to_s == "production"
+        ["#digital-general"]
+      else
+        ["#{}digital-technical"]
```

Comment:
What do these braces mean? It seems like that would negate the presence of the `#`.

Reply:
bug/typo, yup. Wonder how it was working anyway? Will remove.

Reply:
Editor being too clever?